README.rst

```diff
@@ -28,7 +28,9 @@ The manual for Dwarf Therapist can be downloaded separately.
 Building
 ========
 
-Dwarf-Therapist requires a C++ compiler (with C++14 support), cmake (3.1.0 or newer), and Qt5 (with Widgets and QML modules).
+Dwarf-Therapist requires a C++ compiler (with C++14 support), cmake (3.1.0 or newer), and Qt5.5 (with Widgets and QML modules).
```

Comment:
Qt 5.5 is not required. Any Qt 5 version should work. If DT use some features that was added during the Qt 5 cycle, it should be "Qt 5.x or newer". I have built DT on Ubuntu Trusty with Qt 5.2 so it is at least that.

Reply:
News to me, I'll update